tests/boot/grub_test_snapshot.pm

```diff
@@ -13,13 +13,15 @@ use base 'opensusebasetest';
 use testapi;
 use power_action_utils 'power_action';
 use utils qw(workaround_type_encrypted_passphrase reconnect_mgmt_console);
-use bootloader_setup qw(stop_grub_timeout boot_into_snapshot);
+use bootloader_setup qw(stop_grub_timeout boot_into_snapshot change_grub_config);
 use Utils::Backends 'is_pvm';
 
 sub run {
     my $self = shift;
 
     select_console 'root-console';
+    # disable GRUB_TIMEOUT to avoid this module failed since grub menu timeout
+    change_grub_config('=.*', '=-1', 'GRUB_TIMEOUT', '', 1);
```

Comment:
if only happen on aarch64, we can add "if arch==aarch64."

Reply:
I agree with @coolgw, on the other hand would it do any harm if we disable the timeout on all architectures?
If we don't probe for a specific architecture and disable grub unconditionally, then we should think if the line with `stop_grub_timeout ` later in the test module is still needed.

Reply:
yes, you are right, I agree with you, I have improved the code.